plugins/pulp_docker/plugins/distributors/publish_steps.py

```diff
@@ -91,19 +94,30 @@ def initialize(self):
                                                     self.get_repo())
         self.redirect_context.initialize()
 
-    def process_unit(self, unit):
+    def get_count(self):
```

Comment:
I assume this is overriding something in the base class, and that something in the platform calls it.

Simple as it is, a doc block would be valuable. Hopefully you can just copy the one from the platform.